### DISCUSSION TITLE
Dropdowns

### DISCUSSION THREAD

**Discussion by Manalalafad** [TDRA-ae/aegov-dls#39] [Ideas]
While frameworks typically cover dropdown functionality, it is crucial to note that for dropdown lists containing nationalities, the first choice should always be "United Arab Emirates."

**Comment by moiz-conceptlz** [comment]
Hello @Manalalafad 

This is a good suggestion. Let's add this to "Patterns" under the - https://designsystem.gov.ae/docs/patterns and explain the pattern of how to accept Nationality.

**Comment by zowber** [comment]
Many products and website that request nationalities with a dropdown often group the most commonly nations items at the top of the list.

It suggest it's worth considering recommending ministries and authorities look at the data about which options are selected most often and use this information to group options alphabetically at the top of the list. Perhaps, in the case where this data isn't readily available, falling back to the top 5-10 nationalities present in the country?
